src/build_context.rs

```diff
@@ -166,16 +166,45 @@ impl BuildContext {
         }
     }
 
-    /// For abi3 we only need to build a single wheel and we don't even need a python interpreter
-    /// for it
-    pub fn build_binding_wheel_abi3(&self, major: u8, min_minor: u8) -> Result<PathBuf> {
-        // On windows, we have picked an interpreter to set the location of python.lib,
-        // otherwise it's none
-        let artifact = self.compile_cdylib(self.interpreter.get(0), Some(&self.module_name))?;
+    #[cfg(feature = "auditwheel")]
+    fn auditwheel(
+        &self,
+        python_interpreter: Option<&PythonInterpreter>,
+        artifact: &Path,
+        manylinux: &Manylinux,
+    ) -> Result<Option<Policy>> {
+        if !self.skip_auditwheel {
+            let target = python_interpreter
+                .map(|x| &x.target)
+                .unwrap_or(&self.target);
 
-        let platform = self
-            .target
-            .get_platform_tag(&self.manylinux, self.universal2);
+            let policy = auditwheel_rs(&artifact, target, manylinux)
+                .context(format!("Failed to ensure {} compliance", manylinux))?;
+            Ok(policy)
+        } else {
+            Ok(None)
+        }
+    }
+
+    #[cfg(not(feature = "auditwheel"))]
```

Comment:
Now that manylinux has become such an essential feature, we could also drop that feature flag